.github/CODEOWNERS

```diff
@@ -126,11 +125,9 @@
 # Contributor Dashboard project.
 /core/controllers/contributor_dashboard*.py @oppia/contributor-experience-team
 /core/domain/opportunity*.py @oppia/contributor-experience-team
-# TODO(#13808): Revert ownership to @vojtechjelinek after 2021-09-14.
 /core/domain/translation*.py @oppia/contributor-experience-team
 /core/domain/voiceover_services*.py @oppia/contributor-experience-team
-# TODO(#13808): Revert ownership to @vojtechjelinek after 2021-09-14.
```

Comment:
Can you please update the PR title and description so that this issue gets closed once the PR is merged?